src/locales/br/translation.json

```
@@ -0,0 +1,278 @@
+    "download_options": "Opções de download",
+    "download_path": "Diretório de download",
+    "previous_screenshot": "Captura de tela anterior",
+    "next_screenshot": "Próxima captura de tela",
+    "screenshot": "Captura de tela {{number}}",
+    "open_screenshot": "Ver captura de tela {{number}}",
+    "download_settings": "Ajustes do download",
+    "downloader": "Downloader",
+    "select_executable": "Explorar",
+    "no_executable_selected": "Nenhum executável selecionado",
+    "open_folder": "Abrir pasta",
+    "open_download_location": "Ver arquivos baixados",
+    "create_shortcut": "Criar atalho na área de trabalho",
+    "remove_files": "Remover arquivos",
+    "options": "Gerenciar",
+    "remove_from_library_description": "Isso irá remover {{game}} da sua biblioteca",
+    "remove_from_library_title": "Tem certeza?",
+    "executable_section_title": "Executável",
+    "executable_section_description": "O caminho do arquivo que será executado ao clicar em \"Jogar\"",
+    "downloads_secion_title": "Downloads",
+    "downloads_section_description": "Confira atualizações ou versões diferentes para este mesmo título",
+    "danger_zone_section_title": "Zona de perigo",
+    "danger_zone_section_description": "Remova o jogo da sua biblioteca ou os arquivos que foram baixados pelo Hydra",
+    "download_in_progress": "Download em andamento",
+    "download_paused": "Download pausado",
+    "last_downloaded_option": "Última opção baixada",
+    "create_shortcut_success": "Atalho criado com sucesso",
+    "create_shortcut_error": "Erro ao criar atalho"
+  },
+  "activation": {
+    "title": "Ativação",
+    "installation_id": "ID da instalação:",
+    "enter_activation_code": "Insira seu código de ativação",
+    "message": "Se você não sabe onde conseguir o código, talvez você não devesse estar aqui.",
+    "activate": "Ativar",
+    "loading": "Carregando…"
+  },
+  "downloads": {
+    "resume": "Retomar",
+    "pause": "Pausar",
+    "eta": "Conclusão {{eta}}",
+    "paused": "Pausado",
+    "verifying": "Verificando…",
+    "completed": "Concluído",
+    "removed": "Cancelado",
+    "cancel": "Cancelar",
+    "filter": "Filtrar jogos baixados",
+    "remove": "Remover",
+    "downloading_metadata": "Baixando metadados…",
+    "delete": "Remover instalador",
+    "delete_modal_description": "Isso removerá todos os arquivos de instalação do seu computador",
+    "delete_modal_title": "Tem certeza?",
+    "deleting": "Excluindo instalador…",
+    "install": "Instalar",
+    "download_in_progress": "Baixando agora",
+    "queued_downloads": "Na fila",
+    "downloads_completed": "Completo",
+    "queued": "Na fila",
+    "no_downloads_title": "Nada por aqui…",
+    "no_downloads_description": "Você ainda não baixou nada pelo Hydra, mas nunca é tarde para começar.",
+    "checking_files": "Verificando arquivos…"
+  },
+  "settings": {
+    "downloads_path": "Diretório dos downloads",
+    "change": "Explorar...",
```

```suggestion
    "change": "Alterar",
```
English says `Update` but I think this is a file picked so `Explorar...` is good.